neuroscout_cli/cli.py

```diff
@@ -1,76 +1,82 @@
 """
-neuroscout
-
-Usage:
-    neuroscout run [-mfuvd -i <dir> -w <dir> -s <k> -c <n> -n <nv> -e <es>] <outdir> <bundle_id>...
-    neuroscout install [-ui <dir>] <outdir> <bundle_id>...
-    neuroscout upload [-f -n <nv>] <outdir> <bundle_id>...
-    neuroscout ls <bundle_id>
-    neuroscout -h | --help
-    neuroscout --version
-
-Options:
-    -i, --install-dir <dir>  Optional directory to cache input images
-    -w, --work-dir <dir>     Optional Fitlins working directory 
-    -c, --n-cpus <n>         Maximum number of threads across all processes
-                             [default: 1]
-    -s, --smoothing <k>      Smoothing to apply in format: FWHM:level:type.
-                             See fitlins documentation for more information.
-                             [default: 4:Dataset:iso]
-    -u, --unlock             Unlock datalad dataset
-    -n, --neurovault <nv>    Upload mode (disable, all, or group)
-                             [default: group]
-    -e, --estimator <es>     Estimator to use for first-level model
-                             [default: afni]
-    -f, --force-neurovault   Force upload, if a NV collection already exists
-    -m, --drop-missing       If contrast is missing in a run, skip.
-    -d, --no-datalad-drop    Don't drop DataLad sourcedata. True by default
-                             if install-dir is specified.
-    -v, --verbose	         Verbose mode
-    
-
-Commands:
-    run                      Runs analysis.
-    install                  Installs a bundle and/or dataset.
```

Comment:
coming at this naively, it feels a little weird that when I `install` I could specify two paths, one for the dataset and one for the bundle, i don't know if this should be split into two different commands so one could install the dataset (only) or install the bundle (only). looks like the current options are:
1. install bundle
2. install bundle + data

For me I was expecting a third option:
1. install bundle
2. install data (new)
3. install both
That helped conceptually clear up that the install command can install two separate things in two separate directories.

but perhaps this is because I did not read the documentation closely enough.

Reply:
@jdkent I think the real answer is you _need_ the analysis bundle to download the data. So currently it's not possible to only download the data